### PR TITLE
Code Added

### PR DESCRIPTION
**Pull request by friedcucumber**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,121 @@
-# maui-grid-column-chooser
+<!-- default badges list -->
+<!-- default badges end -->
+## DataGrid for .NET MAUI - Implement a Column Chooser
+
+This example shows how to implement a [Data Grid](https://docs.devexpress.com/MAUI/403255/data-grid/data-grid) control's column chooser.
```
Comment (dmitry-eliseev-devexpress) [inline README.md:5]: You wrote the following in the corresponding documentation topic:

> This example demonstrates how to implement a @DevExpress.Maui.DataGrid.DataGridView column chooser.

Why this difference? Normally, I would expect this introduction to fall in line with your mapping documentation topic.

Also, shows -> demonstrates

File: README.md [hunk 2]
```diff
@@ -1 +1,121 @@
...
+
+<img src="https://user-images.githubusercontent.com/12169834/228222481-197a4064-f461-4f13-8877-412c81263fd2.png" width="30%"/>
+
+Used controls and their properties:
```
Comment (dmitry-eliseev-devexpress) [inline README.md:9]: As I mentioned before, we don't deal in "used", that is, second-hand controls.

```suggestion
Included controls and their properties:
```

File: README.md [hunk 3]
```diff
@@ -1 +1,121 @@
...
+* [CheckEdit](https://docs.devexpress.com/MAUI/DevExpress.Maui.Editors.CheckEdit): [IsChecked](https://docs.devexpress.com/MAUI/DevExpress.Maui.Editors.CheckEdit.IsChecked), [Label](https://docs.devexpress.com/MAUI/DevExpress.Maui.Editors.CheckEdit.Label), [CheckBoxPosition](https://docs.devexpress.com/MAUI/DevExpress.Maui.Editors.CheckEdit.CheckBoxPosition)
+
+
+## Implementation Details
+
+* [Data Grid](https://docs.devexpress.com/MAUI/403255/data-grid/data-grid)'s column chooser is displayed within the [DXPopup](https://docs.devexpress.com/MAUI/DevExpress.Maui.Controls.DXPopup) control. [DXPopup](https://docs.devexpress.com/MAUI/DevExpress.Maui.Controls.DXPopup) contains the [DXCollectionView](https://docs.devexpress.com/MAUI/DevExpress.Maui.CollectionView.DXCollectionView) that is bound to the [DataGridView.Column](https://docs.devexpress.com/MAUI/DevExpress.Maui.DataGrid.DataGridView.Columns) collection:
```
Comment (dmitry-eliseev-devexpress) [inline README.md:19]: I explained these issues and the corresponding changes at least in one of the previous merge requests in the series.

```suggestion
* A [Data Grid](https://docs.devexpress.com/MAUI/403255/data-grid/data-grid) column chooser is displayed in the [DXPopup](https://docs.devexpress.com/MAUI/DevExpress.Maui.Controls.DXPopup) control. [DXPopup](https://docs.devexpress.com/MAUI/DevExpress.Maui.Controls.DXPopup) contains the [DXCollectionView](https://docs.devexpress.com/MAUI/DevExpress.Maui.CollectionView.DXCollectionView) bound to the [DataGridView.Column](https://docs.devexpress.com/MAUI/DevExpress.Maui.DataGrid.DataGridView.Columns) collection:
```

File: README.md [hunk 4]
```diff
@@ -1 +1,121 @@
...
+                    </Grid>
+                </DataTemplate>
+            </dxcv:DXCollectionView.ItemTemplate>
+        </dxcv:DXCollectionView>
+    </dxco:DXPopup>
+    ```
+
+    File to Look At: [MainPage.xaml](CS/MainPage.xaml)
+
+* Set the [DXCollectionView.ReduceSizeToContent](https://docs.devexpress.com/MAUI/DevExpress.Maui.CollectionView.DXCollectionView.ReduceSizeToContent) property to `true` to size the [DXCollectionView](https://docs.devexpress.com/MAUI/DevExpress.Maui.CollectionView.DXCollectionView) to fit its content:
```
Comment (dmitry-eliseev-devexpress) [inline README.md:39]: Yes, in theory, you can use "size" in meaning of "resize", however, "resize" sounds more precise in this scenario.

```suggestion
* Set the [DXCollectionView.ReduceSizeToContent](https://docs.devexpress.com/MAUI/DevExpress.Maui.CollectionView.DXCollectionView.ReduceSizeToContent) property to `true` to resize the [DXCollectionView](https://docs.devexpress.com/MAUI/DevExpress.Maui.CollectionView.DXCollectionView) to fit its content:
```

File: README.md [hunk 5]
```diff
@@ -1 +1,121 @@
...
+
+    ```xaml
+    <dxcv:DXCollectionView ReduceSizeToContent="True" ...>
+        <!-- ... -->
+    </dxcv:DXCollectionView>
+    ```
+
+    File to Look At: [MainPage.xaml](CS/MainPage.xaml)
+
+* To align the [CheckEdit](https://docs.devexpress.com/MAUI/DevExpress.Maui.Editors.CheckEdit)'s check boxes to right, set the [CheckEdit.CheckBoxPosition](https://docs.devexpress.com/MAUI/DevExpress.Maui.Editors.CheckEdit.CheckBoxPosition) property to [CheckBoxPosition.End](https://docs.devexpress.com/MAUI/DevExpress.Maui.Editors.CheckBoxPosition.End):
```
Comment (dmitry-eliseev-devexpress) [inline README.md:49]: **the** "CheckEdit" what? It's impossible to interpret this proper noun as a simple or compound noun. Look -- to align the check edit? I would have added a simple noun as the type name. For example:

> To align the CheckEdit component's check boxes ...

Then...

> to right

In this context, "right" is an abstract direction, you can even probably feel that it sounds "incorrect". A definite article is in order...

The rest looks fine in this paragraph.

```suggestion
* To align the [CheckEdit](https://docs.devexpress.com/MAUI/DevExpress.Maui.Editors.CheckEdit) component's check boxes to the right, set the [CheckEdit.CheckBoxPosition](https://docs.devexpress.com/MAUI/DevExpress.Maui.Editors.CheckEdit.CheckBoxPosition) property to [CheckBoxPosition.End](https://docs.devexpress.com/MAUI/DevExpress.Maui.Editors.CheckBoxPosition.End):
```

File: README.md [hunk 6]
```diff
@@ -1 +1,121 @@
...
+                    </Border>
+                </Grid>
+            </DataTemplate>
+        </dxcv:DXCollectionView.ItemTemplate>
+    </dxcv:DXCollectionView>
+    ```
+
+    File to Look At: [MainPage.xaml](CS/MainPage.xaml)
+
+* You can bind the [CheckEdit](https://docs.devexpress.com/MAUI/DevExpress.Maui.Editors.CheckEdit) to the [GridColumn.ActualCaption](https://docs.devexpress.com/MAUI/DevExpress.Maui.DataGrid.GridColumn.ActualCaption) property to display the [Data Grid](https://docs.devexpress.com/MAUI/403255/data-grid/data-grid)'s column caption:
```
Comment (dmitry-eliseev-devexpress) [inline README.md:67]: Again, the proper noun "CheckEdit" does not work as a simple noun. I would have just removed an article in this case.

```suggestion
* You can bind [CheckEdit](https://docs.devexpress.com/MAUI/DevExpress.Maui.Editors.CheckEdit) to the [GridColumn.ActualCaption](https://docs.devexpress.com/MAUI/DevExpress.Maui.DataGrid.GridColumn.ActualCaption) property to display the [Data Grid](https://docs.devexpress.com/MAUI/403255/data-grid/data-grid)'s column caption:
```

File: README.md [hunk 7]
```diff
@@ -1 +1,121 @@
...
+                    </Border>
+                </Grid>
+            </DataTemplate>
+        </dxcv:DXCollectionView.ItemTemplate>
+    </dxcv:DXCollectionView>
+    ```
+
+    File to Look At: [MainPage.xaml](CS/MainPage.xaml)
+
+* Specify the [GridColumn.MinWidth](https://docs.devexpress.com/MAUI/DevExpress.Maui.DataGrid.GridColumn.MinWidth) property to hide horizontal scroll bar when the summary width of visible [Data Grid](https://docs.devexpress.com/MAUI/403255/data-grid/data-grid) columns is lower than the width of the device view port:
```
Comment (dmitry-eliseev-devexpress) [inline README.md:85]: > to hide horizontal scroll bar

Well, scroll bar is countable, therefore, an article is in order.

```suggestion
* Specify the [GridColumn.MinWidth](https://docs.devexpress.com/MAUI/DevExpress.Maui.DataGrid.GridColumn.MinWidth) property to a hide horizontal scroll bar when the summary width of visible [Data Grid](https://docs.devexpress.com/MAUI/403255/data-grid/data-grid) columns is lower than the width of the device view port:
```

**Review by dmitry-eliseev-devexpress — COMMENTED** [review]
I fixed a number of article cases in this pull request as well as added the same changes I made in previous merge and pull requests in the series. Please examine the article cases closely to avoid such mistakes in the future. Thanks in advance.

I'll approve this pull request once we fix the introductory sentence so it falls in line with the corresponding introduction in documentation.

File: README.md (hunk 1 above)
Comment (friedcucumber) [reply README.md:5]:
```suggestion
This example demonstrates how to implement a [Data Grid](https://docs.devexpress.com/MAUI/403255/data-grid/data-grid) column chooser.
```

Comment (dmitry-eliseev-devexpress) [reply README.md]: Good call. Thank you.

**Review by dmitry-eliseev-devexpress — APPROVED** [review]

File: README.md [hunk 8]
```diff
@@ -1 +1,121 @@
...
+
+    ```xaml
+    <dxcv:DXCollectionView ReduceSizeToContent="True" ...>
+        <!-- ... -->
+    </dxcv:DXCollectionView>
+    ```
+
+    File to Look At: [MainPage.xaml](CS/MainPage.xaml)
+
+* To align the [CheckEdit](https://docs.devexpress.com/MAUI/DevExpress.Maui.Editors.CheckEdit) component's check boxes to the right, set the [CheckEdit.CheckBoxPosition](https://docs.devexpress.com/MAUI/DevExpress.Maui.Editors.CheckEdit.CheckBoxPosition) property to [CheckBoxPosition.End](https://docs.devexpress.com/MAUI/DevExpress.Maui.Editors.CheckBoxPosition.End):
```
Comment (dirk-pieterse) [inline README.md:49]:
```suggestion
* To align the [CheckEdit](https://docs.devexpress.com/MAUI/DevExpress.Maui.Editors.CheckEdit) component's checkboxes to the right, set the [CheckEdit.CheckBoxPosition](https://docs.devexpress.com/MAUI/DevExpress.Maui.Editors.CheckEdit.CheckBoxPosition) property to [CheckBoxPosition.End](https://docs.devexpress.com/MAUI/DevExpress.Maui.Editors.CheckBoxPosition.End):
```

File: README.md [hunk 9]
```diff
@@ -1 +1,121 @@
...
+                    </Border>
+                </Grid>
+            </DataTemplate>
+        </dxcv:DXCollectionView.ItemTemplate>
+    </dxcv:DXCollectionView>
+    ```
+
+    File to Look At: [MainPage.xaml](CS/MainPage.xaml)
+
+* Specify the [GridColumn.MinWidth](https://docs.devexpress.com/MAUI/DevExpress.Maui.DataGrid.GridColumn.MinWidth) property to a hide horizontal scroll bar when the summary width of visible [Data Grid](https://docs.devexpress.com/MAUI/403255/data-grid/data-grid) columns is lower than the width of the device view port:
```
Comment (dirk-pieterse) [inline README.md:85]:
```suggestion
* Specify the [GridColumn.MinWidth](https://docs.devexpress.com/MAUI/DevExpress.Maui.DataGrid.GridColumn.MinWidth) property to a hide horizontal scroll bar when the total width of visible [Data Grid](https://docs.devexpress.com/MAUI/403255/data-grid/data-grid) columns is less than the width of the device's viewport:
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**